app/experimental_appconfig.go

```diff
@@ -73,13 +73,45 @@ func setCustomModules(app *RegenApp, interfaceRegistry types.InterfaceRegistry)
 }
 
 func (app *RegenApp) registerUpgradeHandlers() {
-	app.UpgradeKeeper.SetUpgradeHandler("v0.43.0-beta1-upgrade", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-
-		// skipping x/auth migrations. It is already patched in regen-ledger v1.0
-		fromVM["auth"] = auth.AppModule{}.ConsensusVersion()
+	app.UpgradeKeeper.SetUpgradeHandler("v0.43.0-beta1-upgrade", func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+		// 1st-time running in-store migrations, using 1 as fromVersion to
```

Comment:
btw, shouldn't all this be copied to stable_appconfig.go too? As I see,
- stable: v043, authz, feegrant
- experimental: above + group, data, ecocredit

Reply:
Ah okay. I thought we are still keeping authz in expermental. Yes, let's move them to stable config

Reply:
we should also create an issue to remove this upgrade handler before tagging rc-1